### PR TITLE
budgs loadout bloat - colorable flats

### PR DESCRIPTION
**Pull request by BulletBudgie**
+Colorable flats, found in loadout

-Removes bloat code from tajara loadout

all sprites by me

### CODE REVIEW

**Comment by BulletBudgie** [comment]
!review

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
If you want to make tajaran dress flats recolorable you must do it for the normal dress flats as well since they are coded to be an in-tandem system - tajaran flats inherit majority from them and are modified on equip

**Comment by BulletBudgie** [comment]
> If you want to make tajaran dress flats recolorable you must do it for the normal dress flats as well since they are coded to be an in-tandem system - tajaran flats inherit majority from them and are modified on equip

? im only doing the normal dress flats. the other bit is just removing vestigial code because, like you said, flats are modified on equip for tajara - making the xenowear selection unnecessary, changes arent needed.

File: code/modules/clothing/shoes/xeno/tajara.dm [hunk 1]
```diff
@@ -17,42 +17,7 @@
...
-	name = "purple dress flats"
-	icon_state = "purpledf"
-	item_state = "purpledf"
-
-/obj/item/clothing/shoes/flats/tajara/white
-	desc = "A pair of white women's flats. Refitted for Tajara."
-	name = "white dress flats"
-	icon_state = "whitedf"
-	item_state = "whitedf"
+// removed at request of cyb
```
Comment (NonQueueingMatt) [inline code/modules/clothing/shoes/xeno/tajara.dm:20]: remove this comment

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**